### ISSUE TITLE
more compressors

### ISSUE THREAD

**Issue by french-gamer** [MineMaarten/PneumaticCraft#363]
Please add more compressors so that the people who don't have extra mods like buidcraft or industrialcraft can produce air pressure easier.

**Comment by MineMaarten** [comment]
I agree. the difficulty with this is that I want to try and only add compressors which are creative in their way of generating power. Suggestions are appreciated. Even then I couldn't get around implementing the usual power generators, so even if you don't have BC/IC2/Cofh, you still have a solid fuel compressor and a Liquid Compressor :), and the admittedly 

Again, suggestions are appreciated :)

**Comment by french-gamer** [comment]
You could make a Manuel one we you have to run on a treadmill and it will make a small amount of pressure

**Comment by iTitus** [comment]
Handpump, like Forestry's Clockwork Engine, which explodes, when the pressier is too high...

**Comment by DarkStarDS9** [comment]
Hydro-compressor. Has to be placed next to flowing water. Tracks the flow back to the source block to calculate the height difference - more height difference = more effective. Must have some cool animation, of course.
May even be feasible to check for a dam lake - for even more efficiency.
